core/vm/settings.go

```diff
@@ -17,9 +17,9 @@
 package vm
 
 var (
-	DisableJit  bool // Disable the JIT VM
-	ForceJit    bool // Force the JIT, skip byte VM
-	MaxProgSize int  // Max cache size for JIT Programs
+	DisableJit  bool = true // Disable the JIT VM
```

Comment:
We should consider changing the name away from the negative, i.g. `EnableJit bool = false`

Turning a flag on to disable a feature is conceptually strange